### ISSUE TITLE
[Suggestion] Auto Eating

### ISSUE THREAD

**Issue by Lightcaster5** [cabaletta/baritone#496]
So I was using Baritone (the mod) for MineCraft version 1.12.2 and while traveling on the nether highway in 2b2t there was no auto eating function. My idea is to have a few new settings shown below, all defaults are suggestions.

foodLevelToEat ([type : int],   default 10,   this is half total hunger lost)
delayBeforeAutoEat ([type : double/float],   in seconds,   default 0.5,   the delay between Baritone seeing that food level is below {foodLevelToEat} and eating)
delayBetweenChecks ([type : int],  in seconds,  how long before Baritone checks the food level again)
moveFoodInInventory ([type : boolean],   default false,   allows Baritone to move food from inventory to hotbar)

I hope my suggestion will be added and thanks for even taking the time to read this. Thanks leijurv for creating the wonderful AI, Baritone.

**Comment by Vap0r1ze** [comment]
Impact has this

**Comment by Lightcaster5** [comment]
> Impact has this

Yes I understand which is why I specified the forge standalone mod not the implements with impact

**Comment by MrDodojo** [comment]
Still impacts auto eat doesnt work with the miner

**Comment by S-B99** [comment]
Impacts autoeat now works with miner but having a standalone autoeat is still needed.

**Comment by JamesTheAwesomeDude** [comment]
Wurst's AutoEat _doesn't_ interact well with Baritone, so this would definitely be appreciated from this side of the fence!

**Comment by l1ving** [comment]
That is a wurst issue

**Comment by dravpin** [comment]
i guess not

**Comment by Crafter-Y** [comment]
is there any other autoeat mod, that has good compatibility with baritone?

**Comment by Random-Spawn** [comment]
can we get this added please?

**Comment by Yuang114514** [comment]
> still nothing?

yeah
